py3status/modules/deadbeef.py

```diff
@@ -25,20 +27,29 @@
 Requires:
     deadbeef: a GTK+ audio player for GNU/Linux
 
-@author mrt-prodz
+@author mrt-prodz, tobes, lasers
 """
 
-import subprocess
+from subprocess import check_output
+
+CONTROL_PARAMETERS = ['%title%', '%isplaying%']
+CONTROL_SEPARATOR = u''
+FORMAT_SEPARATOR = u''
```

Comment:
I think they're 0x1E and 0x1D. Am I doing it wrong? Record separator and group separator.